python/GafferUI/CompoundEditor.py

```diff
@@ -1655,7 +1655,7 @@ def getToolTip( self ) :
 		toolTipElements = []
 
 		if masterEditor is not None :
-			toolTipElements.append( "Following _%s_" % masterEditor.getTitle() )
+			toolTipElements.append( "Following _%s_\n" % masterEditor.getTitle() )
```

Comment:
Unrelated to this change, but related to the tooltip generally...

 In playing with the linking, I've found it confusing that the swatch and the pinning icon are different widgets, with different tooltips and different clickability. Given that they're both providing a representation of the same thing, it feels like they should share all the same interaction patterns, and you should be able to click the swatch to break the link and popup the menu. The linking is quite fiddly anyway, and requiring folks to hover the swatch for status but use the separate button for editing makes it a fair bit harder to interact with.

Reply:
Agreed. I don't really like the whole thing, but conscious of rabbit holes given the other priorities of late. How much time do we want to spend on this?

Reply:
Maybe we should poll some users? I'm pretty sure though, that if we don't do it now, we won't do it for a long time...